routes.py

```diff
@@ -27,4 +50,4 @@ def get_tasks():
 
 if __name__ == '__main__':
     # Threaded option to enable multiple instances for multiple user access support
-    app.run(threaded=True, port=5000)
+    app.run(debug=True, threaded=True, port=5000 ,use_reloader=False)
```

Comment:
You can pass these arguments from CMD when running flask/app. that way you dont have to modify it again and again